pysys/xml/project.py

```diff
@@ -342,6 +344,27 @@ def _parseClassAndConfigDict(self, node, defaultClass):
 		cls = getattr(module, classname)
 		return cls, optionsDict
 
+def getProjectTemplates():
```

Comment:
This is reading the templates from a directory within the PySys installation, correct?  Do we want to allow for a case where the user running PySys can't write to that directory but wants to use some custom template - or indeed the case that an extension package, e.g. Apama, wants to provide its own project template?  I guess I am suggesting that there should be a list of directories that templates could be taken from.

Reply:
Correct. Unlike test creation (where user-defined tempaltes would be helpful), project creation is so rare that I don't think it's useful to provide a user-facing template capability (there's nowhere obvious to store them anyway), but redistributors like apama can drop extra xml files into the directory easily enough. If there's demand for it we could extend this later but I suspect that's unlikely